Response-Errors.md

```diff
@@ -0,0 +1,123 @@
+- [Not Found Errors](#3-not-found-errors)
+- [Validation Errors](#4-validation-errors)
+- [Global Errors](#5-global-errors)
+- [Payment Errors](#6-payment-errors)
+- [Vendor Errors](#7-vendor-errors)
+- [Organization Errors](#8-organization-errors)
+- [User Errors](#9-user-errors)
+
+### 0. Error Code Legend
+
+Each error code follows a specific format
+
+- First Digit: Group of error
+- Second Digit: Sub group of error
+- Third Digit: If the error is due to **request parameters** or due to configuration 
+- Fourth Digit: Sequent error number
+
+
+### 1. Authorization Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `AUTHORIZATION_ERROR` | `1011` | Token verification failed. Token is not provided  | You are required to provide Lime Pay token for authorization where it is applicable | Accessing resources which requires token authorization |
+| `AUTHORIZATION_ERROR` | `1012` | Token verification failed. Some part is missing | You will get this error if you provide us wrong token structure | Accessing resources which requires token authorization |
+| `AUTHORIZATION_ERROR` | `1013` | Token verification failed. Invalid algorithm | You will get this error if provided token is calculated with wrong algorithm | Accessing resources which requires token authorization |
+| `AUTHORIZATION_ERROR` | `1014` | Failed to verify token | You will get this error if the structure of the token is valid, but it is not actually Lime Pay token | Accessing resources which requires token authorization |
+| `AUTHORIZATION_ERROR` | `1111` | Unauthorized request. Invalid API user | You will get this error if your api user authentication fails | Accessing resources which requires api user authorization |
+| `AUTHORIZATION_ERROR` | `1112` | Unauthorized request. Invalid User | You will get this error if your user authentication fails | Accessing resources which requires user authorization |
+
+### 2. Blockchain Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |----------- | ----------- | ---------- |
+| `BLOCKCHAIN_ERROR` | `2001` | Insufficient token amount in escrow contract |  You will get this error if vendor's escrow contract does not have enough tokens to fund a shopper | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2002` | Insufficient wei amount in escrow contract | You will get this error if vendor's escrow contract does not have enough wei to fund a shopper | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2111` | Invalid gas limit | You will get this error if signed transactions gas limit is different that the relevant one provided with generic transactions | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2112` | Invalid receiver | You will get this error if signed transactions `to` addresses is different from the `to` addresses provided with generic transactions | Processing payment |
```

Comment:
`You will get this error if signed transactions to addresses is different from the to addresses provided with generic transactions`
->
`You will get this error if the `to` address of a signed transaction does not match the provided `to` address on payment creation`